src/controllers/simplified-account/settings/card-payments/card-payments.controller.js

```diff
@@ -26,6 +26,7 @@ function get (req, res) {
     applePayLink: formatSimplifiedAccountPathsFor(paths.simplifiedAccount.settings.cardPayments.applePay, service.externalId, account.type),
     googlePayEnabled: account?.allowGooglePay,
     googlePayLink: formatSimplifiedAccountPathsFor(paths.simplifiedAccount.settings.cardPayments.googlePay, service.externalId, account.type),
+    googlePayEditable: account.getActiveCredential() !== null,
```

Comment:
Do the card-payments.controller.test.js not need to be updated with this true/false field?

Reply:
covered in a separate PR